src/tools/rest-backend/create_entries.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+for i in $(seq 100 150)
```

Comment:
Unimportant script to create some dummy data, I think it can be removed or replaced by some more useful test data as you suggested.